panedr/panedr.py

```diff
@@ -421,11 +421,28 @@ def edr_to_df(path, verbose=False):
             all_energies.append([frame.t] + [ener.e for ener in frame.ener])
 
     end = time.time()
-    if verbose:
+    if verbose_set:
         print('\rLast Frame read : {}, time : {} ps'
               .format(ifr, frame.t),
               end='', file=sys.stderr)
         print('\n{} frame read in {:.2f} seconds'.format(ifr, end - begin),
               file=sys.stderr)
+
+    return all_energies, all_names, times
+
+
+def edr_to_df(path: str, verbose: bool = False):
+    import pandas
+    all_energies, all_names, times = read_edr(path, verbose_set=verbose)
     df = pandas.DataFrame(all_energies, columns=all_names, index=times)
     return df
+
+
+def edr_to_dict(path: str, verbose: bool = False):
+    import numpy as np
+    all_energies, all_names, times = read_edr(path, verbose_set=verbose)
+    energy_dict = {}
+    for idx, name in enumerate(all_names):
+        energy_dict[name] = np.array(
+            [all_energies[frame][idx] for frame in range(len(times))])
+    return energy_dict
```

Comment:
Make sure that the "Time" key is in. I expect it to be, but I do not remember exactly how I treated it.

Reply:
"Time" is part of `all_names` and `all_energies` as returned by `read_edr`. This is done specifically by 

```python
all_names = [u'Time'] + [nm.name for nm in edr_file.nms]
[...]
all_energies.append([frame.t] + [ener.e for ener in frame.ener])
```